AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/BundleHelper.java

```diff
@@ -172,6 +173,10 @@ public BundleHelper(AndroidAnnotationsEnvironment environment, TypeMirror elemen
 			if (isTypeParcelable(type)) {
 				methodNameToSave = "put" + "Parcelable";
 				methodNameToRestore = "get" + "Parcelable";
+			} else if (annotationHelper.isAnnotatedWith(type, "org.parceler.Parcel")) {
```

Comment:
`Parcels.wrap/unwrap` method [can handle](http://stackoverflow.com/questions/29674199/use-android-annotations-with-parceler/29680080#comment47641892_29691052) `List<AnnotatedWithParcel>` AFAIK, or even more. We should also allow those.